tests/integration_tests/performance/test_block_ab.py

```diff
@@ -240,3 +247,104 @@ def test_block_vhost_user_performance(
             **vm.dimensions,
         }
     )
+
+
+def _check_block_size(ssh_connection, dev_path, size):
+    """
+    Checks the size of the block device.
+    """
+    _, stdout, stderr = ssh_connection.run("blockdev --getsize64 {}".format(dev_path))
+    assert stderr == ""
+    assert stdout.strip() == str(size)
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs, vcpu_count, metrics):
+    """
+    This test simply tries to boot a VM with vhost-user-block
+    as a root device and upload FirecrackerMetrics emitted by
+    the test to Cloudwatch.
+    """
+
+    orig_size = 10  # MB
+    new_sizes = [20, 10, 30]  # MB
```

Comment:
Should these be a parametrization of this test?

Also, nit: Can we sort them? e.g. [10, 20, 30]

Reply:
I just copy pasted them from another test to experiment with them and see something in CW.
But good point, I will arrange this when I clean up.

Reply:
on second thought I think the sequence is intentional to test the effect of resize for low->high->low->high configs. I'll double confirm this.